### ISSUE TITLE
lambda arm64

### ISSUE THREAD

**Issue by moshfrid** [serverless-nextjs/serverless-next.js#2134]
Is there any way to run the lambda functions on the arm64 lambda architecture?

**Comment by thiskevinwang** [comment]
This is not an answer to your question, but it looks like [Lambda@Edge](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/edge-functions-restrictions.html) doesn't yet support arm64 architecture.

So, `defaultNextLambda`, `nextApiLambda`, `nextImageLambda` (all lambda @ edge) won't be able to be run on Arm64 arch.

The only _regular_ lambda is the `regenerationFunction` for ISR, which could potentially run on Arm64

**Comment by moshfrid** [comment]
I see the restriction in the link you provided, however, weirdly enough I was able to simply switch the architecture on the lambda@edge image optimization function and its working correctly (and faster).

**Comment by thiskevinwang** [comment]
@moshfrid 💭 you might be seeing a false positive? 🤔 

Given that it's an Edge lambda, you'd need to deploy a new version. (But I wouldn't recommend doing so manually, since all this is managed through CDK/CloudFormation. You don't want any "drift")

![image](https://user-images.githubusercontent.com/26389321/144085037-ac7a0554-334d-495f-ab2c-cf261bf55cc8.png)

**Comment by moshfrid** [comment]
You are absolutely correct, my bad, still would be nice to be able to take advantage of the arm architecture for the regeneration lambda.
